src/next/azext_next/_client_factory.py

```diff
@@ -0,0 +1,8 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def cf_next(cli_ctx, *_):  # pylint: disable=unused-argument
```

Comment:
Do we need this factory as it just returns None?

Reply:
done